src/de/willuhn/jameica/hbci/gui/dialogs/ExportDialog.java

```diff
@@ -25,8 +25,6 @@
 import org.eclipse.swt.widgets.FileDialog;
 
 import de.willuhn.datasource.GenericObject;
-import de.willuhn.datasource.pseudo.PseudoIterator;
```

Comment:
nicht mehr notwendig wegen [Änderung in Zeile 320 (jetzt Zeile 305)](https://github.com/willuhn/hibiscus/pull/98/files#diff-453aa339164180732b0c800a9696c9bae3d864769c93344aa2d105d3745c5e01R305)